apps/api-key-service/src/constants/session.ts

```diff
@@ -0,0 +1,23 @@
+import { type IronSession, type SessionOptions } from 'iron-session'
+
+import { envVars } from './envVars'
+
+export const sessionOptions: SessionOptions = {
+  password: envVars.IRON_SESSION_SECRET,
+  cookieName: `api-key-service.${envVars.DEPLOYMENT_ENV}.auth.session-token`,
```

Comment:
we can probably remove all the cookie related logic, if we need anything on the admin api side of things we'll just proxy it thru the dapp console api to this service

Reply:
updated to remove all admin api related code